testutil/singlevm/setup.sh

```diff
@@ -142,22 +208,34 @@ then
 fi
 
 #Generate Certificates
-"$GOPATH"/bin/ciao-cert -server -role scheduler -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
+"$GOPATH"/bin/ciao-cert -server -role scheduler -email="$ciao_email" \
+    -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
 
-"$GOPATH"/bin/ciao-cert -role cnciagent -server-cert "$ciao_cert" -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
+"$GOPATH"/bin/ciao-cert -role cnciagent -server-cert "$ciao_cert" \
+    -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
 
-"$GOPATH"/bin/ciao-cert -role controller -server-cert "$ciao_cert" -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
+"$GOPATH"/bin/ciao-cert -role controller -server-cert "$ciao_cert" \
+    -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
 
-"$GOPATH"/bin/ciao-cert -role agent,netagent -server-cert "$ciao_cert" -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
+"$GOPATH"/bin/ciao-cert -role agent,netagent -server-cert "$ciao_cert" \
+    -email="$ciao_email" -organization="$ciao_org" -host="$ciao_host" -ip="$ciao_ip" -verify
 
-openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout controller_key.pem -out controller_cert.pem -subj "/C=US/ST=CA/L=Santa Clara/O=ciao/CN=$ciao_host"
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
+    -keyout ${ciao_controller_key} -out ${ciao_controller_cert} -subj "/C=US/ST=CA/L=Santa Clara/O=ciao/CN=$ciao_host"
 
 #Copy the certs
-sudo cp -f controller_key.pem /etc/pki/ciao
-sudo cp -f controller_cert.pem /etc/pki/ciao
-sudo ln -s /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
-sudo ln -s /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
-
+sudo cp -f ${ciao_controller_key} ${ciao_pki_path}
+sudo cp -f ${ciao_controller_cert} ${ciao_pki_path}
+sudo ln -s ${ciao_pki_path}/${ciao_controller_key} ${ciao_pki_path}/${ciao_image_key}
+sudo ln -s ${ciao_pki_path}/${ciao_controller_cert} ${ciao_pki_path}/${ciao_image_cert}
+#Update system's trusted certificates
+CACERT_PROG=$(type -p update-ca-certificates)
```

Comment:
I see in the commit comments that this was done to get ciao-image to start.   I wonder if there's a way to get ciao-image to work without using the system certs.  It currently starts okay using the fake identity service using just CIAO_CA_CERT_FILE.  What problems were you seeing?